scripts/occlum-host-test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+ANONIFY_ROOT="$(cd $(dirname $0); pwd)/.."
+
+i=0
+MAX_TRIES=40
+# Wait until the enclave is healthy
+set +e
+while true;
+do
+    grpcurl -plaintext -d '{ "service": "helloworld.Greeter" }' "$OCCLUM_ENCLAVE_IP_ADDRESS:$OCCLUM_ENCLAVE_PORT" grpc.health.v1.Health/Check
+    if [ $? -eq 0 ]; then
+        echo "enclave is ready"
+        break;
+    fi
+
+    ((i=i+1))
+    if [ $i -gt $MAX_TRIES ]; then
+        echo "The number of trials has exceeded the maximum." >&2
+        exit 1
+    fi
+
+    echo "Tried health check to the enclave $i times..."
+    sleep 10
```

Comment:
40秒で現実的に起動することが見込めるなら 

```suggestion
    sleep 1
```

くらいにしておきたいです 😴

Reply:
40はcounterなので、 40*10秒 です。（実際は20*10sくらい）、CIのログ荒れる && CI全体のレイテンシーから数秒の遅延は無視できるのでこのままにしておきたいと思うのですがどうでしょ

Reply:
あ、sleep 1 にしたならば40秒になるねという意図でした。

ヘルスチェック通るまで200秒はかかるということでしょうか？そうだとすると今の設定値で全然妥当だと思います。

Reply:
あー、です。enclaveのcargo buildとか待っておるので